understanding/21/reflow.html

```diff
@@ -12,68 +12,235 @@ <h1>Understanding Reflow</h1>
+        
+        <p>Thus, while following responsive web design best practices will help many websites meet Reflow, the success criterion does not mandate fully responsive web pages. There will be instances in which more complex web interfaces need to expend additional effort by instead providing revised presentation, functionality, or both when content is enlarged - and thus presented in smaller viewport sizes. In other cases, aspects of the application's content will meet one of Reflow's exceptions and may be presented in a bi-directional scrolling container within the otherwise single-direction scrolling interface.</p>
+
+        <h4>Horizontally scrolling section within a vertically scrolling page</h4>
+        <p>It is not uncommon for a web page to render a widget, such as a "carousel", "swim lane", or otherwise "horizontally scrolling container" which itself contains sub-sections of content that follow the otherwise esbalished reading direction of the page. Such widgets can meet the requirements of this success criterion, so long as the sub-sections of content can accommodate reflowing to each fit within the required width or height - per the intended reading direction each section represents.</p>
+
+        <figure>
+          <img src="img/carousel-large-screen.png" alt="horizontally scrolling carousel, containing multiple 'panels' of content (four visible in the screenshot).">
+          <figcaption>
+            <p>A website provides a “carousel” or “swim-lane” of different panels of teaser (a.k.a., “lead-in”) content.This content container allows for horizontal scrolling, independent from the vertical scrolling of the primary web page, and the vertical direction of reading of the individual sections of content within this horizontally scrolling container.</p>
+          </figcaption>
+        </figure>
+
+        <figure>
+          <img src="img/single-carousel-panel.png" alt="one of the carousel panels zoomed in 400%. The text vertically flowing text fits within 320px width container." width="550">
+          <figcaption>
+            <p>The content of each carousel "panel" (image, text content and call to action link visually represented as a right-pointing arrow icon) fit within a width of 320 CSS pixels. This helps ensure a user need only scroll vertically to read the text of each section of teaser content. The user can horizontally scroll the wrapping container element to bring each width-conforming panel of teaser content into view.  A user can scroll vertically to view the image or read the text of each section of content.</p>
```

Comment:
This is great; I think it'd be even better if it also included a corresponding example of a similar failing case (where an individual carousel item didn't fit). It might be helpful to use a consistent format for marking which examples pass and which fail (eg, in https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance.html, we had the figcaptions start with "Passes: " and "Fails: " for the figures demonstrating individual examples)